md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/SubProcessAPI.java

```diff
@@ -130,16 +130,20 @@ public RestWrapper get(
     public RestWrapper delete(
             @PathVariable("id") Integer processId, Principal principal) {
         RestWrapper restWrapper = null;
-        try {
-            com.wipro.ats.bdre.md.dao.jpa.Process parentProcess=processDAO.get(processId);
-            processDAO.securityCheck(parentProcess.getProcessId(),principal.getName(),WRITE);
+       try {
+            LOGGER.info("going to delete process id : " + processId);
+            com.wipro.ats.bdre.md.dao.jpa.Process process=processDAO.get(processId);
```

Comment:
@Prem1993 delete all unnecessary logger statements, if you have put them for debugging please remove them.